apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -780,13 +848,19 @@ def test(element_html, data):
         )
         answer.pop(0)
         score = 0
-        if grading_mode == "unordered" or (
-            grading_mode in ["dag", "ranking"] and partial_credit_type == "lcs"
+        if grading_method is GradingMethodType.UNORDERED or (
```

Comment:
I think that checking `grading_method in ...` should still work. Will shorten this boolean check.